dc-cudami-editor/src/components/FeedbackMessage.jsx

```diff
@@ -24,6 +28,7 @@ const FeedbackMessage = ({className = 'mb-0', message, onClose}) => {
           <FaTimes size={16} />
         </Button>
       )}
+      <Trans components={links} i18nKey={`feedback:${key}`} values={values} />
```

Comment:
Aus welchem Grund hast du hier die Reihenfolge geändert?

Reply:
Weil ich in der Fehlermeldung eine Liste hab (rein Unicode, lediglich mit `<br>`-Tags) und dann das Kreuzchen unten rechts stehen würde. So bleibt's oben, egal wieviel Inhalt kommt.